0232. Implementing Queue using Stack/MyQueue.java

```diff
@@ -0,0 +1,56 @@
+class MyQueue {
+    /** Two stacks to maintain the queue functionoing */
+    private Stack<Integer> s1 = new Stack<>();
+    private Stack<Integer> s2 = new Stack<>();
+
+    /** Initialize your data structure here. */
+    public MyQueue() {
+    }
+
```

Comment:
Looks Great, just nit picks.
Better to declare stack s1 and s2 above constructor and initialize them using the constructor.
you can consider moving new stack<>() inside constructor.